src/app/contextBroker/contextBroker.cpp

```diff
@@ -176,6 +176,8 @@
 #include "common/string.h"
```

Comment:
Entry in CHANGES_NEXT_RELEASE missing (I understand that referring to issue number https://github.com/telefonicaid/fiware-orion/issues/956)